src/institutions/Refile.jsx

```diff
@@ -1,14 +1,11 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import RefileButton from '../refileButton/container.jsx'
-import * as STATUS from '../constants/statusCodes.js'
+import { PARSED_WITH_ERRORS, VALIDATING } from '../constants/statusCodes.js'
```

Comment:
I've done this in a few places just to make what is actually being used more clear.